app/scripts/modules/core/src/naming/moniker.model.ts

```diff
@@ -0,0 +1,8 @@
+
+export class Moniker {
```

Comment:
This should be an interface, `IMoniker`. We're avoiding using explicit classes for domain objects unless we're adding methods to them.

(should also remove the leading newline character)

Reply:
@anotherchrisberry - sounds good, I changed it.